addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -457,6 +457,12 @@ img::selection {
             padding: 0.5rem 0; // To use more editor space if necessary
         }
     }
+
+    &.o_fullscreen {
+        .modal:not(.o_technical_modal) {
+            right: 0;
+        }
+    }
```

Comment:
From what I gather from the code, the o_fullscreen class is a website added one, so its related rules should not be in web_editor